src/HW4_featureEng.py

```diff
@@ -178,7 +178,7 @@ def main():
     for a, b in zip(continuous, sorted(continuous, reverse=True)):
         test = Cat_vs_Cont(a, b).contResponse_vs_contPredictor()
```

Comment:
looks like you have already merged your pR for HW4...

Missing table on all variables and ranking
missing Random Forest importance
missing mean of responses

I am assuming you are still working on it